skietbaan_fe/src/scss/view-comp.css

```diff
@@ -379,7 +379,6 @@
 }
 
 .total-header-label {
-
 	height: 100%;
```

Comment:
indent your code[highlight the code , right click then select 'Format Document' ]